backend/app/api/endpoints/openapi_responses.py

```
@@ -321,9 +328,288 @@ async def create_response(
+            output=_build_output(subtasks, "in_progress"),
+            previous_response_id=request_body.previous_response_id,
+        )
+
+    # SSE mode with background=False: sync wait for completion
+    _close_db()
+    try:
+        accumulated_content = await _execute_and_collect()
+        logger.info(f"[OPENAPI] Sync completed for subtask {assistant_subtask_id}")
+        await _handle_completion(accumulated_content)
+    except Exception as e:
+        logger.exception(f"Error in sync chat response: {e}")
```

Ruff (0.15.5)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)